src/components/shared/forms/tooltip.js

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import React, { Component } from 'react';
+
+import { joinClasses } from 'utilities';
+
+import './styles/tooltip.css';
+
+export class Tooltip extends Component {
+  render() {
+    const { target, children, position } = this.props;
+
+    return (
+      <div class="tooltip-container">
```

Comment:
These should be className

Reply:
Well heck, React isn't doing us any favors, is it?  It used to not work if you messed that up.